lib/geokit/geocoders/bing.rb

```diff
@@ -65,7 +65,7 @@ def self.set_address_components(loc, xml)
       ACCURACY_MAP = {
         'High'   => 8,
         'Medium' => 5,
-        'Low'    => 2
+        'Low'    => 2,
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.